### PR TITLE
The great pethat spritefest

### PR DESCRIPTION
**Pull request by kit-katz**
## About The Pull Request
Introducing PET HATS. QUITE A LOT OF EM
also a few pets can now be picked up

adds holding and or head sprites for:

* almost all corgi variants + a pug
* the 2 corgi puppies (one with new handsprites)
* ALL cat variants
* poly the slipery (if you manage to grab him)
* captain's fox for there is only 1
* mice and hamsters
* jani's pet lizard
* chicks(the OTHER kind)
* Crabs
* SNAKES!!
all coded to be on your head... obviously
not that i made the code. and if i knew who it was i would give them credit... THANKS

imageimageimageimageimageimageimageimageimageAnimation

imageimageimageimageimageimageimageimageimageimage
and plenty more (mostly corgi and cat resprites) ^

i take no criticism for anything. the sprites exausted me. if you think you can do better be my guest

## Why It's Good For The Game
have you ever missed your pet that is just locked in your room. or just noticed that all the pets other than ian are barely seen by human and not so human eyes. OR just stuck in your backpack
INTRODUCING HAT PETS. the luster of your favorite pet now with you for all to see

also given the oportunity to give traitors the ability to throw angry snakes at their target over just bombing medical for the 1000th time YES ILL TAKE SAID OPORTUNITY.

the framework was there i just took an oportunity. i doubt cosmetic changes would be a problem. 
unless there is some pet meta im not aware of

## Changelog
:cl:
add: plenty of pets and critters can now be picked up and worn as a hat (ethically)
add: most corgi types,pugs, corgi puppies, Walter, ALL cat types(yes even cak), cap's fox for there is only 1, jani's pet lizard
add: poly can now be picked up and worn as a hat but good luck with that
add: mice, hamsters, chickens and chicks(The other type) crabs and SNAKES!!
imageadd: sprites for the above
/:cl:

### CODE REVIEW

**Comment by ItsTisme** [comment]
CAT HAT CAT HAT CAT HAT CAT HAT

**Comment by kit-katz** [comment]
i just had a PAI cat... i put it on as a hat... and i had a spiritual awakening

**Comment by imsynthex** [comment]
cat hat cat hat cat hat. good PR speedmerge now

**Comment by Archanial** [comment]
What about Walter?

**Comment by OceanFish1** [comment]
What about hat version of Lamarr

**Comment by Raven-Industries** [comment]
Does this mean we can wear big rat as a hat?

**Comment by kit-katz** [comment]
sadly walter is to big for a hat. a hat sprite can be 5 pixels tall 6-7 if you start overlaping the head of the player

o shit lamar is a good idea - sadly lammar and any face hugger are clasified as masks. without invasive coding it cannot be done

it will still lool like a notrmal rat size wise... but im pretty sure admins can MAKE IT BIGGER

**Comment by Archanial** [comment]
How come corgi can fit and walter doesn't 😢

**Comment by francinum** [comment]
> sadly walter is to big for a hat. a hat sprite can be 5 pixels tall 6-7 if you start overlaping the head of the player
> 
> o shit lamar is a good idea - sadly lammar and any face hugger are clasified as masks. without invasive coding it cannot be done
> 
> it will still lool like a notrmal rat size wise... but im pretty sure admins can MAKE IT BIGGER

This is not necessarily true, see the speedwagon hat, you would just need a specific icon file for the oversized pets.

**Comment by kit-katz** [comment]
I DONT EVEN HAVE TO TOUCH ANY CODE AND I BREAK SOMETHING !!!

**Comment by kit-katz** [comment]
long overdue due to personal reasons. and a walter exorsism. its here
Walter hat and... chicken

imageimage

i'm officially done... for now

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -315,7 +315,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	force = 2
 	throwforce = 20 //20 + 2 (WEIGHT_CLASS_SMALL) * 4 (EMBEDDED_IMPACT_PAIN_MULTIPLIER) = 28 damage on hit due to guaranteed embedding
 	throw_speed = 4
-	embedding = list("embedded_pain_multiplier" = 4, "embed_chance" = 100, "embedded_fall_chance" = 0)
+	embedding = list("embedded_pain_multiplier" = 1, "embed_chance" = 100, "embedded_fall_chance" = 0)
```
Comment (Archanial) [inline code/game/objects/items/weaponry.dm]: how is this related to pethats?

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 2]
```diff
@@ -38,8 +39,8 @@
 	icon_state = "mouse_[body_color]"
 	icon_living = "mouse_[body_color]"
 	icon_dead = "mouse_[body_color]_dead"
-	if(prob(40))
-		var/datum/disease/advance/R = new /datum/disease/advance/random(rand(2, 4))
+	if(prob(75))
+		var/datum/disease/advance/R = new /datum/disease/advance/random(rand(3, 6), 9, rand(3,4))
```
Comment (Archanial) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm]: same here

File: code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (kit-katz) [reply code/game/objects/items/weaponry.dm]: its... not where did this come from???

File: code/modules/mob/living/simple_animal/friendly/mouse.dm (hunk 2 above)
Comment (kit-katz) [reply code/modules/mob/living/simple_animal/friendly/mouse.dm]: ok this one tho. if i revert this one gitbot will scream merge conflict.
¯\_(ツ)_/¯

File: code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (Archanial) [reply code/game/objects/items/weaponry.dm]: I don't know, you commited this

Comment (kit-katz) [reply code/game/objects/items/weaponry.dm]: oh right it's a future commit. must have acidentally ploped it to this branch and it added it. compleatly forgot about it

**Comment by ItsTisme** [comment]
How is this coming along?

**Comment by kit-katz** [comment]
the PR itself... its done
the github... PAIN on a spaghetti plate

**Comment by kit-katz** [comment]
fuck the mice i guess. o well
my stupidity and spagetti chef skills have reached full circule and fixed a merge conflict.
